client/src/components/Query/Statement.jsx

```diff
@@ -428,6 +451,10 @@ class Statement extends React.Component {
             categories= {categories}
             onCopyCallback={this.handleCopy}
             onEditCallback={this.handleEdit}
+            onAddInstructionCallback={this.handleAddInstruction}
+            onRemoveInstructionCallback={this.handleRemoveInstruction}
+            onReplaceInstructionCallback={this.handleReplaceInstruction}
+            onChangeQueryTitleCallback={this.handleChangeQueryTitle}
```

Comment:
Are these not all `onEditCallback` ?

Reply:
No because, for example, you don't want to trigger an API call because you simply change the Title of the Query. Think of the redux sagas.

Reply:
`onAddInstructionCallback`, `onRemoveInstructionCallback` and `onReplaceInstructionCallback are similar to `onEditCallback`. Can we either keep `onEditCallback` or the other ones? Or is `onEditCallback` use for the display options?

Reply:
We want to know the difference between the old ```onEditCallback``` and the four new following one
- onAddInstructionCallback
- onRemoveInstructionCallback
- onReplaceInstructionCallback
- onChangeQueryTitleCallback